godtools/App/Share/Data/UserAuthentication/AuthenticationProvider/Response/AuthenticationProviderResponse+MobileContentAuthProviderToken.swift

```diff
@@ -21,11 +21,18 @@ extension AuthenticationProviderResponse {
             
         case .apple:
             
-            guard let idToken = self.idToken, !idToken.isEmpty else {
-                return .failure(NSError.errorWithDescription(description: "Missing apple idToken."))
+            if let authCode = self.appleSignInAuthorizationCode, authCode.isEmpty == false {
+                
+                return .success(.appleAuth(authCode: authCode, givenName: profile.givenName, familyName: profile.familyName))
+                
+            } else if let refreshToken = self.refreshToken, refreshToken.isEmpty == false {
+                
+                return .success(.appleRefresh(refreshToken: refreshToken))
```

Comment:
Hey @rachaelblue just realized something that this refreshToken portion will never evaluate to anything since the Apple refreshToken isn't part of the provider.